src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -184,7 +184,7 @@ const formConfig = {
                 'This is the military service history we have on file for you.',
               'ui:options': {
                 itemName: 'Service Period',
-                viewField: ServicePeriodView,
+                viewField: ValidatedServicePeriodView,
```

Comment:
@bshyong I added this inside 526EZ, too. Should this live in both places or just the one?

Reply:
Only all-claims is active now for new users. This change means that people are are resuming a V1 form will see the changes as well which should be ok